packages/test-app/webpack.config.js

```diff
@@ -33,6 +34,10 @@ const TestAppFederation = new ModuleFederationPlugin({
         singleton: true,
         eager: true,
       },
+      '@openshift/dynamic-plugin-sdk': {
```

Comment:
:+1: on adding the `@openshift/dynamic-plugin-sdk` to the shared scope.

However, note that webpack federated module sharing works based on matching module specifiers.

So this will cause a plugin to use the application provided shared module version, as expected:
```ts
import { stuff } from '@openshift/dynamic-plugin-sdk';
```
while this will cause the plugin provided fallback module version (if shipped along with the plugin) to be used, because there's no matching entry in the shared scope:
```ts
import { stuff } from '@openshift/dynamic-plugin-sdk/abc';
```

It is the host application's responsibility to initialize and maintain its shared scope object, and to communicate information about application provided modules to its plugins.

Reply:
I'll put this as todo to create a function to match all exposed modules as well.